packages/elastic_agent/data_stream/cloud_defend_logs/fields/fields.yml

```
@@ -2,8 +2,10 @@
   type: text
 - name: input
   type: object
+  object_type: object
```

I guess we only need `input.type`.
```suggestion
- name: input.type
  description: "Input type"
  type: keyword
```